modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -97,4 +94,217 @@ Status description:: error: connection exception - alias chains are not permitte
+
+=== 42N11
+
+Status description:: error: syntax error or access rule violation - database or alias already exists. A [composite] database or alias with the name { $db } already exists.
+
+=== 42N12
+
+Status description:: error: syntax error or access rule violation - user already exists. A user with the name { $user } already exists.
+
+=== 42N13
+
+Status description:: error: syntax error or access rule violation - role already exists. A role with the name { $role } already exists.
+
+=== 42N14
+
+Status description:: error: syntax error or access rule violation - invalid use of command. { $clause } cannot be used together with `$cmd`.
+
+=== 42N15
+
+Status description:: error: syntax error or access rule violation - invalid use of reserved keyword. { $syntax } is a reserved keyword and cannot be used in this place.
+
+=== 42N16
+
+Status description:: error: syntax error or access rule violation - unsupported index or constraint. Only single property { $idxType } are supported.
+
+=== 42N17
+
+Status description:: error: syntax error or access rule violation - unsupported request. { $input } is not allowed on the system database.
+
+=== 42N49
+
+Status description:: error: syntax error or access rule violation - unsupported normal form. Unknown Normal Form: { $input }.
+
+=== 42N75
+
+Status description:: error: syntax error or access rule violation - invalid use of graph function. A call to the graph function { $fun } is only allowed as the top-level argument of a USE clause.
+
+=== 42N83
+
+Status description:: error: syntax error or access rule violation - impersonation disallowed while password change required. Cannot impersonate a user while password change required.
+
+=== 42N84
+
+Status description:: error: syntax error or access rule violation - TERMINATE TRANSACTION misses YIELD clause. WHERE clause without YIELD clause. Use 'TERMINATE TRANSACTION ... YIELD ... WHERE ...'.
+
+=== 42N85
+
+Status description:: error: syntax error or access rule violation - cannot specify both allowed and denied databases. Allowed and denied database options are mutually exclusive.
+
+=== 42N88
+
+Status description:: error: syntax error or access rule violation - cannot grant privilege. Permission cannot be granted for 'REMOVE IMMUTABLE PRIVILEGE'.
+
+=== 42N89
+
+Status description:: error: syntax error or access rule violation - invalid driver settings map. Failed evaluating the given driver settings. { $cause }
+
+=== 42N90
+
+Status description:: error: syntax error or access rule violation - cannot alter immutable composite database. Composite databases cannot be altered (database: { $db }).
+
+=== 42I50
+
+Status description:: error: syntax error or access rule violation - token name too long. Invalid input { $input }. A { $tokenType } name cannot be longer than { $maxTokenLength }.
```

Comment:
If the order should be alpha-numeric this one should be before 42N.. right?

Reply:
Also in the implementation I can see that the message is 
`Invalid input { %s }... A { %s } name cannot be longer than { %s }.` Probably the '...' is a bug in the codebase and the message should be like you have written it here, but I will ask Emelie from kernel that implemented this one to confirm

Reply:
The input that is printed is truncated and after discussions in the team-error channel on slack we said to add the three dots after to indicate that. We decided to truncate the input since the string is at least 16383 characters long and if we print the whole string it would be hard for the user to read the message. I think we must have forgotten to update the spreadsheet.

Reply:
Thanks, I forgot about that discussion. Then we should update it here and in the spreadsheet to the ... format instead.